test/integration.ts

```diff
@@ -37,32 +51,32 @@ describe('functional tinlake tests', () => {
     this.timeout(50000);
 
     it('count number of loans', async () => {
-      const count = await tinlake.loanCount();
+      const count = await adminTinlake.loanCount();
       console.log(`Found ${count} loans `);
       assert(count.gte(new BN(0)));
     });
 
     it('get a loan', async () => {
-      const res = await tinlake.getLoan(20);
+      const res = await adminTinlake.getLoan(43);
```

Comment:
If we re-deploy the contracts we won't have that many loans. I think it would be better to use something like `count-1`. Maybe also after the first loan is generated to ensure there is at least one.

Reply:
Good point, I changed the order of the tests now so they don't need any seeded data. Makes the tests more interdependent, which is not a good practice in general, but makes our life a bit simpler and does not pollute the loan list even more.

Reply:
I agree. In the long term, the test should not run against kovan.
The idea is to deploy new contracts on ethereum test node and afterwards run independent tests.

If the run tests long-term on kovan, we should also redeploy the contracts